docs/faq.rst

```diff
@@ -0,0 +1,18 @@
+Frequently Asked Questions
+==========================
+
+Why is my image "flipped" when I display it?
+--------------------------------------------
+
+At EuXFEL, the convention is to show detector images looking at the front of the detector, meaning the beam coming from behind you. You can get the other view with EXtra-geom's plotting methods by passing ``frontview=False``.
+
+If you use matplotlib's ``imshow(array)`` to display an image, it may be upside down. You can correct this by passing ``imshow(array, origin='lower')``.
+
+What are "slow_scan" and "fast_scan"?
+-------------------------------------
+
+We talk about "slow_scan" and "fast_scan" rather than just "x" and "y" axes, because the axes of the data array in HDF5 files do not always represent the same physical directions. For example, working along the "fast_scan" axis might be increasing "y" in one AGIPD module, and decreasing "y" in another.
```

Comment:
```suggestion
What are "slow-scan" and "fast-scan"?
-------------------------------------

We talk about "slow-scan" and "fast-scan" rather than just "x" and "y" axes, because the axes of the data array in HDF5 files do not always represent the same physical directions. For example, working along the fast-scan axis might be increasing y in one AGIPD module, and decreasing y in another.
```

`fast_scan` in code, but fast-scan as a description in text, IMO.